server/routes/caseAdmin/index.ts

```diff
@@ -50,8 +50,13 @@ export default function Index({
   const assessmentHandler = new AssessmentRoutes(caseAdminCaseloadService)
   get(paths.prison.assessment.home, assessmentHandler.GET)
 
-  const tasklistRoutes = new TasklistRoutes(eligibilityAndSuitabilityService)
-  get(paths.prison.assessment.initialChecks.tasklist, tasklistRoutes.GET)
+  const eligibilityAndSuitabilityQuestionHandler = new EligibilityAndSuitabilityQuestionHandler(
```

Comment:
Can we call this `*QuestionListHandler` - `QuestionHandler` sounds like a singular question